obs/metrics.go

```diff
@@ -96,3 +101,27 @@ func NewHistogram(name, desc string) Histogram {
 func InitPrometheusMetricsProvider(namespace, subsystem string) {
 	__provider = NewPrometheusMetricsProvider(namespace, subsystem)
 }
+
+type MetricsServerConfig struct {
+	ServiceName string
+	Port        string
+}
+
+const DefaultMetricServerPort = ":8080"
```

Comment:
Create a function called DefaultMetricsServerConfig that builds and returns a default config. Take ServiceName as a param since we can't have a default value for it.